distributed/worker.py

```diff
@@ -1597,8 +1597,6 @@ async def close(  # type: ignore
 
         await self.scheduler.close_rpc()
```

Comment:
This is misleading. This is actually a noop since `Worker.scheduler` is of type `PooledRPCCall` and not `rpc`

```suggestion
```

https://github.com/dask/distributed/blob/eb297b3e7f4d045f80be0a5b50b42a579adc37ee/distributed/core.py#L1348-L1382